### DISCUSSION TITLE
Adjusting kernel in histplot KDE

### DISCUSSION THREAD

**Discussion by aleksejalex** [mwaskom/seaborn#3109] [Q&A] [answered]
Hi, 
I have a problem - there's no mention in the documentation how exactly choose different kernels for KDE (either in histplot or kdeplot). The only info I found mentions that kdeplot uses Gaussian kernel by default.

Is there any way to change the choice of kernel?

**Comment by mwaskom** [comment, marked as answer]
No, `kdeplot` only supports Gaussian kernels.

**Reply by aleksejalex** [reply]
OK, shame. Anyway thank you for quick answer.

**Reply by kateus65** [reply]
Anyway there can be options to support alt kernels- not too keen on using R and/or Stata to do what used to be an option in seaborn?
